donate/settings/base.py

```diff
@@ -37,6 +37,9 @@ class Base(object):
     # Basket Configuration
     BASKET_API_ROOT_URL = env('BASKET_API_ROOT_URL')
     BASKET_SQS_QUEUE_URL = env('BASKET_SQS_QUEUE_URL')
+    # Basket client configuration
+    BASKET_URL = env('BASKET_URL')
+    BASKET_API_KEY = env('BASKET_API_KEY')
```

Comment:
Basket URL is set in the env.default file for the staging basket URL. 

The Basket API Key isn't needed for the standard HTTP request to Basket, but I put it in here for good measure in case we need it in the future.